prepare_cp/roles/omnia_telemetry_cp/files/prerequisite.py

```diff
@@ -67,6 +68,17 @@ def check_amd_gpu_existence():
     else:
         dict_component_existence["amdgpu"] = False
 
+def check_gaudi_existence():
+    '''
+    Method to check whether Gaudi is present
+    '''
+    gaudi_output = invoke_commands.call_command_with_pipe\
+        ("lspci|grep \"Habana Labs Ltd. Device\"")
```

Comment:
Gaudi check line is different here. Can we keep same mentioned in https://github.com/dell/omnia/blob/release_1.7/discovery/roles/postscripts/common/templates/omnia_gaudi.j2#L10